packages/rpc-core/src/rpc-methods/__tests__/get-cluster-nodes-test.ts

```diff
@@ -0,0 +1,27 @@
+import { createHttpTransport, createJsonRpc } from '@solana/rpc-transport';
+import type { Rpc } from '@solana/rpc-transport/dist/types/json-rpc-types';
+import fetchMock from 'jest-fetch-mock-fork';
+
+import { createSolanaRpcApi, SolanaRpcMethods } from '../index';
+
+describe('getClusterNodes', () => {
+    let rpc: Rpc<SolanaRpcMethods>;
+    beforeEach(() => {
+        fetchMock.resetMocks();
+        fetchMock.dontMock();
+        rpc = createJsonRpc<SolanaRpcMethods>({
+            api: createSolanaRpcApi(),
+            transport: createHttpTransport({ url: 'http://127.0.0.1:8899' }),
+        });
+    });
+
+    it('returns the cluster nodes', async () => {
+        expect.assertions(5);
+        const res = await rpc.getClusterNodes().send();
+        expect(res).toBeTruthy();
+        expect(res).toHaveLength(1);
+        expect(res[0].rpc).toBe('127.0.0.1:8899');
+        expect(res[0].gossip).toBe('127.0.0.1:1024');
+        expect(res[0].tpu).toBe('127.0.0.1:1026');
```

Comment:
Slight preference replacing this entire thing with:

```ts
expect(res).toMatchObject(
  expect.arrayContaining({
    gossip: '127.0.0.1:1024',
    rpc: '127.0.0.1:8899',
    tpu: '127.0.0.1:1026',
  }),
));
```

…or whatever the magic incantation is.